Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Building this image will include the running of all benchmarks in a "cold
+# cache" state, and thus will take quite a bit of time (roughly 30-60 minutes,
+# with Ganache excluded).  The cache will be retained in the built image, such
+# that running the image will execute benchmarks with a "warm cache". For
+# example:
```

Comment:
How would you clear the cache if you wanted to? Would be good to add that in the comments here

Reply:
I've got an example command drafted for this, but I'm still verifying it. Will report back in a few hours.

Reply:
Perfect, thanks! Everything else here looks good so will merge once that's added

Reply:
Addressed in https://github.com/mds1/convex-shutdown-simulation/pull/5/commits/565d6f9b049a8128dbfbee86516c2e37b4fcf12b